docs/spec/api.md

```diff
@@ -1,8 +1,13 @@
-<!--GITHUB
```

Comment:
Edits to `api.md` need to made to `api.md.tmpl` and the document should be regenerated.

I'm also a little worried about including this in the documentation. This document is a specification and not a document of how to interface with the registry. We have had some confusion in this space.